Resources/input/fsh/profiles/DispenseInformationBundle.fsh

```diff
@@ -19,16 +19,30 @@ Description: "Communication resource for sending dispenseInformation about a med
 * entry ^slicing.rules = #closed
 * entry contains
     Header 1..1 and
-    DispenseInformation 1..1
+    MedicationRequest 1..1 and
+    DispenseInformation 1..1 and
+    RequestingOrganization 0..1
 * entry[Header].link ..0
 * entry[Header].resource 1..
 * entry[Header].resource only GEM_PR_ERP_MEDREQ_MessageHeaderMedicationRequest
 * entry[Header].search ..0
 * entry[Header].request ..0
 * entry[Header].response ..0
+* entry[MedicationRequest].link ..0
+* entry[MedicationRequest].resource 1..
+* entry[MedicationRequest].resource only KBV_PR_ERP_Prescription
+* entry[MedicationRequest].search ..0
+* entry[MedicationRequest].request ..0
+* entry[MedicationRequest].response ..0
 * entry[DispenseInformation].link ..0
 * entry[DispenseInformation].resource 1..
 * entry[DispenseInformation].resource only GEM_ERP_PR_MedicationDispense
 * entry[DispenseInformation].search ..0
 * entry[DispenseInformation].request ..0
-* entry[DispenseInformation].response ..0
\ No newline at end of file
+* entry[DispenseInformation].response ..0
+* entry[RequestingOrganization].link ..0
```

Comment:
Warum ist die RequestingOrganization in der DispensierInformation wichtig?

Reply:
Dieses Bundle soll ja mal das Business Object "Verordnung_Dispensierung" wiederspiegeln. Die Pflegeeinrichtung kann so im UC "Rezeptanforderung durch Apotheke" hier erstmalig prüfen, ob sie auch gemeint war, weil sie in diesem UC sonst nicht mit dem Rezeptvorgang in Kontakt kommt (nicht mal eine Vorgangsnummer hat), weil alles über die Apotheke läuft. Es sind also Vollständigkeits-/Plausibilisierungsgründe. Habe das Objekt trotzdem auf optional gesetzt, weil es im anderen UC (Pflegeeinrichtung fordert an) kaum relevant ist und dort weggelassen werden könnte.